python/MooseDocs/extensions/MooseTextPatternBase.py

```diff
@@ -109,6 +114,7 @@ def createElement(self, label, content, filename, rel_filename, settings):
         code = etree.SubElement(pre, 'code')
         if settings['language']:
             code.set('class', settings['language'])
-        code.text = self.markdown.htmlStash.store(content.strip('\n'), safe=True)
+        content = content.replace('<', '&lt;').replace('>', '&gt;')
```

Comment:
There is a `cgi.escape` function that comes with python. It only does < > & which are the main ones.